framework/Samplers/Sampler.py

```diff
@@ -578,6 +579,8 @@ def initialize(self,externalSeeding=None,solutionExport=None):
 
     # Register expected metadata
     meta = ['ProbabilityWeight','prefix','PointProbability']
+    for var in self.toBeSampled.keys():
+      meta +=  ['ProbabilityWeight-'+ key for key in var.split(",")]
```

Comment:
I have a question here. If in the sampler, we define variable ```name='x,y'```, will this be properly handled.

Reply:
addressed